data-prepper-plugins/aggregate-processor/README.md

```diff
@@ -75,6 +76,37 @@ While not necessary, a great way to set up the Aggregate Processor [identificati
         { "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "status": 200, "bytes": 1000, "http_verb": "GET" }
       ```
 
+### <a name="count"></a>
+* `count`: Count Events belonging to the same group and generate a new event with values of the identification keys and the count, indicating the number of events. All Events that make up the combined Event will be dropped.
+    * It supports the following config options
+       * `count_key`: key name to use for storing the count, default name is "aggr._count"
+       * `output_format`: format of the aggregated event.
+         * Default is JSON with `count_key` added to the input events with identification keys
+         * Other supported output format is `otel_metrics` which generates OTEL SUM metric with the count as value
```

Comment:
Need to update this documentation now that `otel_metrics` is the default output format and `raw` is non-default